chat2issue/cli.py

```
@@ -0,0 +1,598 @@
+    import urllib.error
+    import urllib.request
+
+    url = f"https://api.github.com/repos/{repo}/issues"
+    data = _json.dumps(payload).encode("utf-8")
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "Authorization": f"Bearer {token}",
+        "User-Agent": "chat2issue-cli",
+        "Content-Type": "application/json",
+    }
+    request = urllib.request.Request(url, data=data, headers=headers, method="POST")
```

Ruff (0.13.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)